docs/spec/staking/definitions and examples.md

```diff
@@ -38,7 +38,7 @@
 
 At the core of the Staking module is the concept of a pool which denotes 
 collection of Atoms contributed by different Atom holders. There are two global
-pools in the Staking module: the bonded pool and unbonded pool. Bonded Atoms 
+pools in the Staking module: the bonded pool and unbonding pool. Bonded Atoms 
```

Comment:
I believe it is really unbonded pool. Unbonding pool is something different that we will also probably introduce.

Reply:
Yeh this is definitely wrong.

We should probably stop pushing to each others branches to avoid these kinds of things @adrianbrink @ebuchman

But calling it "unbonded" is misleading too since it really is bonded it's just not with an active validator.

Reply:
hmmm okay - let's come up with some better language - because in this case it is technically "unbonded" because it's no longer slashable - however it can be rebonded by that candidate if it becomes a validator - It's in Limbo Land